OWNERS_ALIASES

```diff
@@ -44,3 +44,9 @@ aliases:
   gateway-api-gep-reviewers:
     - candita
     - gcs278
+
+  gwctl-approvers:
+    - gwctl-approvers
```

Comment:
wait...

Reply:
🤦, should be fixed now 
![image](https://github.com/kubernetes-sigs/gateway-api/assets/419322/a22156cb-8e1e-4984-9cf2-2d2a5e11b2fd)